### PR TITLE
APP-3353 Define fontsize for Avatar

### PR DESCRIPTION
**Pull request by symphony-annaLlorens**
## Description
- Defined default fontsize to Avatar
- Update storybooks examples

Screenshot 2021-03-23 at 17 34 23

[Jira ticket](https://perzoinc.atlassian.net/browse/APP-3353)

### CODE REVIEW

File: src/components/atoms/_avatar.scss [hunk 1]
```diff
@@ -8,25 +24,31 @@ $avatar-sizes: ("xsmall": 16, "small": 24, "medium":  32, "large":  40, "xlarge"
   align-items: center;
   justify-content: center;
   user-select: none;
-  width: 1em;
-  height: 1em;
-  font-size: toEm(map-get($avatar-sizes, "medium")); //Default size 
+  width: toEm(
+    map-get($avatar-sizes, "medium"),
+    map-get($avatar-sizes, "medium")
```
Comment (symphony-adnane) [inline src/components/atoms/_avatar.scss:29]: This won't work :
You're using as a base the same value, it's like writing 1em
The base you need here is map-get($avatar-font-sizes, "medium")
By the way you need maybe to refactor this get in a variable as it's written several times throughts the file

Comment (symphony-annaLlorens) [reply src/components/atoms/_avatar.scss:29]: done

File: src/components/atoms/_avatar.scss [hunk 2]
```diff
@@ -8,25 +25,33 @@ $avatar-sizes: ("xsmall": 16, "small": 24, "medium":  32, "large":  40, "xlarge"
   align-items: center;
   justify-content: center;
   user-select: none;
-  width: 1em;
-  height: 1em;
-  font-size: toEm(map-get($avatar-sizes, "medium")); //Default size 
+  width: toEm(
+    map-get($avatar-sizes, $default-size),
+    map-get($avatar-font-sizes, $default-size)
```
Comment (symphony-adnane) [inline src/components/atoms/_avatar.scss:30]: why not refactoring map-get($avatar-font-sizes, $default-size)? 
it appears 4 times :D

**Review by symphony-jean-michael — APPROVED** [review]

File: src/components/atoms/_avatar.scss (hunk 1 above)
Comment (symphony-annaLlorens) [reply src/components/atoms/_avatar.scss]: Actually it's ok because the fontsize medium is 14px.

**Review by symphony-adnane — APPROVED** [review]

**Pull request merged**